rpm_spec/subpackages/manageiq-system

```diff
@@ -36,6 +36,7 @@ Requires: openldap-clients
 %{_bindir}/pg_inspector_server.sh
 %{_prefix}/lib/systemd/system/cloud-ds-check.service
 %{_prefix}/lib/systemd/system/evm*
+%{_prefix}/lib/systemd/system/manageiq-initialize.service
```

Comment:
I was pretty sure we had a rule for `manageiq-*` and we were slowly trying to get rid of the `evm*` and `miq*` rules

ok to make this generic here?

Reply:
So we have `manageiq-*` as part of the `manageiq-core-services` rpm.  This is for the core workers and we cant have `manageiq-*` in both places